moped-database/migrations/1654025878154_update_proj_partner_filter/up.sql

```diff
@@ -0,0 +1,97 @@
+            ELSE mp.current_status
+        END AS status_name,
+    string_agg((task_order_filter.value ->> 'display_name'::text), ','::text) AS task_order_name,
+    mp.contractor,
+    mp.purchase_order_number,
+    COALESCE(( SELECT json_agg(features.feature) AS json_agg
+           FROM (moped_proj_components components
+             LEFT JOIN moped_proj_features features ON ((features.project_component_id = components.project_component_id)))
+          WHERE (true AND (components.status_id = 1) AND (components.project_id = mp.project_id))), '{}'::json) AS project_feature,
+    fsl.funding_source_name,
+    ptl.type_name,
+    ( SELECT mpn.project_note
+           FROM moped_proj_notes mpn
+          WHERE ((mpn.project_id = mp.project_id) AND (mpn.project_note_type = 2) AND (mpn.status_id = 1))
+          ORDER BY mpn.date_created DESC
+         LIMIT 1) AS project_note,
+    ( SELECT phases.phase_start
+           FROM moped_proj_phases phases
+          WHERE (true AND (phases.project_id = mp.project_id) AND (phases.phase_name = 'construction'::text))
+          ORDER BY phases.date_added DESC
+         LIMIT 1) AS construction_start_date,
+    ( SELECT phases.phase_end
+           FROM moped_proj_phases phases
+          WHERE (true AND (phases.project_id = mp.project_id) AND (phases.phase_name = 'complete'::text))
+          ORDER BY phases.date_added DESC
+         LIMIT 1) AS completion_end_date,
+    ( SELECT string_agg(concat(users.first_name, ' ', users.last_name), ', '::text) AS string_agg
+           FROM ((moped_proj_personnel personnel
+             JOIN moped_users users ON ((personnel.user_id = users.user_id)))
+             JOIN moped_project_roles roles ON ((personnel.role_id = roles.project_role_id)))
+          WHERE ((1 = 1) AND (roles.project_role_name = 'Inspector'::text) AND (personnel.status_id = 1) AND (personnel.project_id = mp.project_id))
+          GROUP BY personnel.project_id) AS project_inspector,
+    ( SELECT string_agg(concat(users.first_name, ' ', users.last_name), ', '::text) AS string_agg
+           FROM ((moped_proj_personnel personnel
+             JOIN moped_users users ON ((personnel.user_id = users.user_id)))
+             JOIN moped_project_roles roles ON ((personnel.role_id = roles.project_role_id)))
+          WHERE ((1 = 1) AND (roles.project_role_name = 'Designer'::text) AND (personnel.status_id = 1) AND (personnel.project_id = mp.project_id))
+          GROUP BY personnel.project_id) AS project_designer
+   FROM (((((((moped_project mp
+     LEFT JOIN project_person_list_lookup ppll ON ((mp.project_id = ppll.project_id)))
+     LEFT JOIN funding_sources_lookup fsl ON ((fsl.project_id = mp.project_id)))
+     LEFT JOIN project_type_lookup ptl ON ((ptl.project_id = mp.project_id)))
+     LEFT JOIN moped_entity me ON ((me.entity_id = mp.project_sponsor)))
+     LEFT JOIN moped_proj_partners mpp2 ON (((mp.project_id = mpp2.project_id) AND (mpp2.is_deleted = false))))
```

Comment:
This is the only update to the `project_list_view` (checking for partner records with `is_deleted = false` instead of `status_id = 1`)